plugins/woocommerce-admin/client/launch-your-store/settings/slotfill.js

```diff
@@ -26,13 +27,40 @@ const SiteVisibility = () => {
 		comingSoon: initialComingSoon = false,
 		storePagesOnly: initialStorePagesOnly = false,
 		privateLink: initialPrivateLink = false,
+		shareKey,
 	} = useLaunchYourStore();
 	const [ comingSoon, setComingSoon ] = useState( initialComingSoon );
 	const [ storePagesOnly, setStorePagesOnly ] = useState(
 		initialStorePagesOnly
 	);
 	const [ privateLink, setPrivateLink ] = useState( initialPrivateLink );
 
+	const copyLink = __( 'Copy link', 'woocommerce' );
+	const copied = __( 'Copied!', 'woocommerce' );
+	const [ copyLinkText, setCopyLinkText ] = useState( copyLink );
+
+	const getPrivateLink = () => {
+		return (
+			window?.wcSettings?.homeUrl +
+			( storePagesOnly === 'yes' ? 'store?woo-share=' : '?woo-share=' ) +
```

Comment:
> `'store?woo-share='`

The `store` permalink could be changed by the user. I think it might need to be looked up on the PHP side and then added to `window.wcSettings` 🤔 

[Example code from ChatGPT](https://chat.openai.com/share/3671c9ca-7391-4281-8740-9fa3f1fc857d)

Reply:
That's a good point! I'll look into it 👍

Reply:
@adrianduffell Fixed in https://github.com/woocommerce/woocommerce/pull/45915/commits/3c8267f768d6aae7a1015b7ef7e302669dd8cc8c

This commit also preloads woocommerce_share_key, woocommerce_coming_soon, woocommerce_store_pages_only, and woocommerce_private_link to speed up the loading times.

cc @psealock We could possibly replace https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/launch-your-store/use-launch-your-store.js with these preloaded settings.